pkg/manifest/manifest.go

```diff
@@ -676,7 +711,12 @@ func (m *MixinDeclaration) UnmarshalYAML(unmarshal func(interface{}) error) erro
 	}
 
 	for mixinName, config := range mixinWithConfig {
-		m.Name = mixinName
+		name, version, err := extractVersionFromName(mixinName)
```

Comment:
Won't the name and version already have been set at line 690? Is it needed to also set it here?

Reply:
Ah sadly yes it is, there are two different logic branches (I think!) One for if you just specify a mixin name, like:
```yaml
mixins:
  - exec
  - terraform
```

And another if you provide config for the mixin, like:
```yaml
mixins:
  - terraform:
    clientVersion: 1.3.0
```